src/Form/FormControl.jsx

```diff
@@ -71,7 +73,7 @@ const FormControl = React.forwardRef(({
       className={className}
     >
       <RBFormControl
-        as={as}
+        as={hasInputMask?.length ? IMaskInput : as}
```

Comment:
I think you can simplify this to `as={inputMask ? IMaskInput : as}` (because if `inputMask` is an empty string it will be considered falsy)

Reply:
Yeah, you are right. Thanks 💯